src/main/docs/guide/httpServer/sessions.adoc

```diff
@@ -99,33 +99,33 @@ snippet::io.micronaut.docs.session.ShoppingControllerSpec[tags="add", indent=0]
 
 == Using @SessionValue
 
-Rather than explicitly injecting the api:session.Session[] into a controller method, you can instead use ann:session.annotation.SessionValue[]. For example:
+Rather than explicitly injecting the https://micronaut-projects.github.io/micronaut-session/snapshot/api/io/micronaut/session/Session.html[Session] into a controller method, you can instead use https://micronaut-projects.github.io/micronaut-session/snapshot/api/io/micronaut/session/annotation/SessionValue.html[@SessionValue]. For example:
```

Comment:
define `https://micronaut-projects.github.io/micronaut-session/snapshot/api` in https://github.com/micronaut-projects/micronaut-core/blob/4.0.x/gradle.properties and uses it in the asciidoc.